lib/core/src/server/config/styleWebpackRules.js

```diff
@@ -0,0 +1,91 @@
+import getCSSModuleLocalIdent from 'react-dev-utils/getCSSModuleLocalIdent';
+
+// style files regexes
+const cssRegex = /\.css$/;
+const cssModuleRegex = /\.module\.css$/;
+const sassRegex = /\.(scss|sass)$/;
```

Comment:
Do I need to update the `angular-cli_config.js` to now also filter out `/\.(scss|sass)$/`?

https://github.com/storybooks/storybook/blob/80ec5726b68aea0dff1245642a888e43c07daaf3/app/angular/src/server/angular-cli_config.js#L114

Reply:
It should be filtered, yeah. It was supposed to be done in #4343.